pseudonymiser/src/integration-test/kotlin/org/ostelco/pseudonymiser/PseudonymiserTest.kt

```diff
@@ -0,0 +1,51 @@
+package org.ostelco.pseudonymiser
+
+import com.google.gson.JsonParser
+import io.dropwizard.testing.ConfigOverride
+import io.dropwizard.testing.ResourceHelpers
+import io.dropwizard.testing.junit.DropwizardAppRule
+import org.glassfish.jersey.client.JerseyClientBuilder
+import org.junit.ClassRule
+import org.junit.Test
+import java.util.*
+import kotlin.test.assertEquals
+
+/**
+ * Class to do integration testing of  pseudonymiser.
+ */
+class PseudonymiserTest {
+
+    private val msisdn = "4790303333"
+    companion object {
+
+        @JvmField
+        @ClassRule
+        val RULE = DropwizardAppRule(
+                PseudonymiserApplication::class.java,
+                ResourceHelpers.resourceFilePath("config.yaml"),
+                ConfigOverride.config("datastoreType", "emulator"))
+    }
+
+    /**
+     * Test a normal request
+     */
+    @Test
+    fun testPseudonymiser() {
+
+        val response = JerseyClientBuilder().build()
+                ?.target("http://0.0.0.0:${RULE.getLocalPort()}/pseudonymiser/status")
+                ?.request()
+                ?.get()
+        assertEquals(200, response?.status)
+
+//        val customToken = String(response?.readEntity(java.lang.String::class.java)?.toCharArray() ?: CharArray(0)).split(".")
```

Comment:
Delete dead code, or add a comment describing  why the could should be kept even if commented out.